### ISSUE TITLE
preview.catalogue.life: test report

### ISSUE THREAD

**Issue by yroskov** [CatalogueOfLife/data#199]
Tests at https://preview.catalogue.life/

Master view: http://dev4.species.id:9191/col_plus/
taxonomic databases 163
living species 1,858,113 
extinct species 38,292
infraspecific living taxa 159,760 
infraspecific extinct taxa 659 
synonyms 1,698,641 
common names 436,607

- [ ] Cannot find the title of the product. Supposed to be "Catalogue of Life: 2020-10-28", but there is no confirmation of that.

- [ ] Cannot judge a total number of accounted species, synonyms, common names in the product.

- [ ] Cannot find how to cite whole product.

**Comment by yroskov** [comment]
- [ ] Cannot judge how complete are taxa in the Tree comparing with a view in old interface

![image](https://user-images.githubusercontent.com/28574252/97589889-0ef6a500-19cc-11eb-92df-0fd09376142d.png)

![image](https://user-images.githubusercontent.com/28574252/97589960-246bcf00-19cc-11eb-9118-2bbc6808c314.png)

**Comment by mdoering** [comment]
see https://github.com/CatalogueOfLife/portal/issues/12 and https://github.com/CatalogueOfLife/portal/issues/12

**Comment by yroskov** [comment]
- [x] Classification checks: all phyla = OK; sampled Class Revtraviricetes, Phylum Acanthocephala, Class Lycopodiopsida = OK

- [ ] Sometimes, collapsing a tree branch takes substantial time.

- [ ] Very inconvenient Tree scrolling, because there are two scroll bars, one for a tree window (window is too small) and another for website window. 

- [ ] Images above the Tree also creates lot of problems (because of that, Tree window is too small for convenient scrolling)

**Comment by yroskov** [comment]
- [ ] Not an optimal positioning of the searched name. Uneasy to find a name.  
Example: searching for _Trifolium scabrum_ 

Results on my screen:
![image](https://user-images.githubusercontent.com/28574252/97595038-4d429300-19d1-11eb-959e-c7ed4aa06f2d.png)

To find a name, I need to scroll down a website window. Woops!

**Comment by yroskov** [comment]
- [ ] It looks like references are missing with names on Species Page.
Example: Phlegmariurus acerosus (Sw.) B. Øllg.

![image](https://user-images.githubusercontent.com/28574252/97597061-6a786100-19d3-11eb-8e05-9f7056a17568.png)

In master data, every name has references:
![image](https://user-images.githubusercontent.com/28574252/97597259-a1e70d80-19d3-11eb-8053-e676b89eece0.png)

**Comment by yroskov** [comment]
- [ ] Unclear how synonyms are ordered (neither alphabetically, nor around protonyms): 

![image](https://user-images.githubusercontent.com/28574252/97598128-9811da00-19d4-11eb-81a3-c6837c0be5fc.png)

**Comment by yroskov** [comment]
- [ ] There are no credits to CoL or GSD in the classification on Species page:

![image](https://user-images.githubusercontent.com/28574252/97598704-3b62ef00-19d5-11eb-8f9e-b0a460ed8c6c.png)

It suppose to be like that:

![image](https://user-images.githubusercontent.com/28574252/97598948-7bc26d00-19d5-11eb-94bc-411420f48f00.png)

**Comment by yroskov** [comment]
- [ ] There is no "how to cite the source" (bibliographic citation) on Species page.

**Comment by yroskov** [comment]
Database Qualifiers on Species page:
![image](https://user-images.githubusercontent.com/28574252/97599533-1ae76480-19d6-11eb-985c-fc260c52fb72.png)

- [ ] Geographic Coverage is missing among Qualifiers.
- [ ] Unclear what means % and stars (context message is needed).

**Comment by yroskov** [comment]
- [ ] Search. Image above search box makes search very inconvenient. User need scroll window down. 

![image](https://user-images.githubusercontent.com/28574252/97601191-ed031f80-19d7-11eb-8ece-397a03dadb6f.png)

**Comment by yroskov** [comment]
Contributing Dataset page (metadata).

(Not sure that metadata have been already imported in tested release from the master list. (@gdower, could you pls comment)

- [ ] Missing Taxonomic coverage

![image](https://user-images.githubusercontent.com/28574252/97602901-dfe73000-19d9-11eb-904e-453414c12fc5.png)

- [ ] Missing Organisation

![image](https://user-images.githubusercontent.com/28574252/97602992-fb523b00-19d9-11eb-9621-e7fba8da722b.png)

- [ ] Something wrong with citation (supposed to be Hobern D., Gielis C. only once):

![image](https://user-images.githubusercontent.com/28574252/97603090-13c25580-19da-11eb-8a54-d175d51f55a7.png)

**Comment by yroskov** [comment]
- [ ] Incomplete citation: 

![image](https://user-images.githubusercontent.com/28574252/97613298-820d1500-19e6-11eb-993f-684801fc3094.png)

**Comment by yroskov** [comment]
@mdoering @thomasstjerne @chantalhuijbers @olafbanki @gdower:

What's next with CoL of 2020-10-28? I believe, we have finished monthly edition in the Clearinghouse. Plus, my report about data view via preview site is above.

Are you expecting anything else from me and Geoff for releasing CoL of 2020-10-28 in a new portal?

**Comment by mdoering** [comment]
@yroskov I have not looked at content. If you think it is good to be released proper let's do it.
Have you checked (some) metadata?

**Comment by gdower** [comment]
I'm working on the metadata.

**Comment by mdoering** [comment]
well yes, you filed some bugs above. Someone needs to go through them

**Comment by yroskov** [comment]
I have repeated my tests on 2020-11-04 for release of 2020-11-03 at http://preview.catalogue.life/index.html. 

Many of reported problems above remain open. We may postpone fixes in a functionality, but content issues should be fixed before release:

[ ] Missing Taxonomic coverage in all GSDs
![image](https://user-images.githubusercontent.com/28574252/98156909-20ddb980-1e9e-11eb-89c5-e5d59594a4e2.png)

[ ] Geographic Coverage is missing among Qualifiers in some GSDs (for example, ELPT, Pterophoroidea, SF Mantodea, SF Orthoptera, etc.
![image](https://user-images.githubusercontent.com/28574252/98156972-394dd400-1e9e-11eb-8c0e-148bfc68cf26.png)

[ ] Strange situation with a list of authors in ICTV: https://github.com/CatalogueOfLife/data/issues/204
Corrected again (as a patch) in the Clearinghouse.
